accession-commons-monotonic-generator-jpa/src/test/resources/application-test1.properties

```diff
@@ -1 +1,14 @@
 accessioning.monotonic.init.category-id-monotonic-test = 100
+accessioning.monotonic.category-id-monotonic-test.blockSize = 1000
+accessioning.monotonic.category-id-monotonic-test.categoryId = category-id-monotonic-test
+accessioning.monotonic.category-id-monotonic-test.applicationInstanceId = instance1
+accessioning.monotonic.category-id-monotonic-test.nextBlockInterval = 0
+accessioning.monotonic.init.eva = 1
+accessioning.monotonic.eva.blockSize = 5
+accessioning.monotonic.eva.categoryId = eva
+accessioning.monotonic.eva.applicationInstanceId = instance1
+accessioning.monotonic.eva.nextBlockInterval = 5
```

Comment:
Could `blockSize` and `nextBlockInterval` be different? That will reflect a more realistic use case, where our block interval is 10k times greater than the block size. Of course the example doesn't need to be so huge ;)

Reply:
Done with eva_2 and contiguous-block-test category